### ISSUE TITLE
Create a tag/release

### ISSUE THREAD

**Issue by terlar** [klarna-incubator/bec#3]
It would be nice if a tag of the current release was available, for those interested in pinning a specific version.

**Comment by robertoaloi** [comment]
Of course. We are working on integrating this repo with some internal integration test. Once that is sorted, we will create tags and releases regularly.
